static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,148 @@
+# Shared Storage on NFS
+
+In the modern software development environment, teams are working together on
+same dataset to get the results. It became necessary that data is accessible and
+every team member has a same updated dataset. NFS (Network File System) storage
```

Comment:
`NAS (NFS is one common example) is widely ... `

Reply:
we can also mention something like: "Here we would like to show you how to setup a shared cache on NFS, but the same idea applies to any other NAS"